crowbar_framework/app/models/node_object.rb

```diff
@@ -759,6 +759,10 @@ def crowbar=(value)
     @role.default_attributes = value
   end
 
+  def role
```

Comment:
Style/TrivialAccessors: Use `attr_reader` to define trivial reader methods. (https://github.com/bbatsov/ruby-style-guide#attr_family)